package.json

```
@@ -14,6 +14,9 @@
     "depcheck": "depcheck",
     "test:unit": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./src/**/*.test.ts",
     "test:integration": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/**/*.test.ts",
+    "test:e2e": "playwright test --ui",
+    "playwright:install": "playwright install --with-deps",
+    "playwright:test": "playwright test",
```

```suggestion
    "playwright:install": "playwright install --with-deps",
    "test:playwright": "playwright test",
```

I think we should call the execution test:playwright because it is in line with other test types we do, like unit, etc.